contrib/debian/copyright

```diff
@@ -154,7 +154,7 @@ Copyright: 1994, 1995, 1996, 1999, 2000, 2001, 2002, 2004, 2005, 2006, 2007 Free
  2013 Ericsson AB
  2014 AppDynamics Inc.
  2015-2016 Brocade Communications Systems Inc.
-License: LGPL-3+ with ZeroMQ exception
+License: MPL-2.0
```

Comment:
MPL-2.0 is a "[file-based copyleft](https://fossa.com/blog/open-source-software-licenses-101-mozilla-public-license-2-0/)" licence; that is, if you were to change the files that are part of ZeroMQ then you'd have to release them under the same terms, but there's no restriction on linking or combining with code under different licences. I guess that's fine, and is approximately what the exception was doing.

Reply:
I left the `depends/patches/zeromq/*` as covered by the same license as their code, so my patch to fix a warning in their code satisfies this requirement.